src/main/java/com/develhope/spring/vehicle/models/Vehicle.java

```diff
@@ -2,6 +2,7 @@
 
 
 import com.develhope.spring.vehicle.ExcessiveParameterException;
+import com.develhope.spring.vehicle.optionals.models.Abs;
```

Comment:
Perché hai importato la classe se ancora non viene usata?

Reply:
Perchè avevo inserito in vehicle i dati delle variabili, poi mi sono reso conto di aver fatto una cazzata e gli ho eliminati, ma non ho tolto i package. Li tolgo